hazelcast/scripts/cleanDownloaded.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 HAZELCAST_BRANCH="$1"
+HAZELCAST_BRANCH="distributed_scheduled_exec"
```

Comment:
This is only until we merge the hazelcast/hazelcast#9087 - Reason for putting (if I recall correctly) is that the `feature/` prefix on my branch is causing inconsistencies on directory names, thus the script didn't work.